src/core/udb-api.service.js

```diff
@@ -1182,22 +1181,21 @@ function UdbApi(
     };
 
     return $http
-        .put(appConfig.baseUrl + 'roles/' + roleId + /constraints/ + version, updateData, requestOptions)
+        .put(appConfig.baseUrl + 'roles/' + roleId + '/constraints/v3', updateData, requestOptions)
```

Comment:
Nit: Is there a way that we could also remove these version numbers from the endpoints? We're the only ones using those for roles management (we don't have public docs for them at least and don't support them for external parties), so in theory we could make a breaking change? Or keep this old endpoint but also add some kind of alias without the version number?

Reply:
I'll do that in a follow-up to align the changes with `udb3-silex`